src/gating/gating.ps1

```diff
@@ -0,0 +1,66 @@
+$pass = ${env:MAPPED_ADO_PAT} #$env:MAPPED_ADO_PAT = "TODO-ADVSEC-SCOPED-PAT-HERE"
+$orgUri = ${env:SYSTEM_COLLECTIONURI} #$(System.CollectionUri) #$env:SYSTEM_COLLECTIONURI = "https://dev.azure.com/TODO-YOUR-ORG-HERE/"
+$orgName = $orgUri -replace "^https://dev.azure.com/|/$"
+$project = ${env:SYSTEM_TEAMPROJECT} #$(System.TeamProject) #$env:SYSTEM_TEAMPROJECT = "TODO-YOUR-PROJECT-NAME-HERE"
+$repositoryId = ${env:BUILD_REPOSITORY_ID} #$(Build.Repository.ID) #env:BUILD_REPOSITORY_ID= "TODO-YOUR-REPO-GUID-HERE"
+$pair = ":${pass}"
+$bytes = [System.Text.Encoding]::ASCII.GetBytes($pair)
+$base64 = [System.Convert]::ToBase64String($bytes)
+$basicAuthValue = "Basic $base64"
+$headers = @{ Authorization = $basicAuthValue }
+$url = "https://advsec.dev.azure.com/{0}/{1}/_apis/AdvancedSecurity/Repositories/{2}/alerts?useDatabaseProvider=true" -f $orgName, $project, $repositoryId
+
+$alerts = Invoke-WebRequest -Uri $url -Headers $headers -Method Get
+if ($alerts.StatusCode -ne 200) {
+    Write-Information "##vso[task.logissue type=error] Error getting alerts from Azure DevOps Advanced Security:", $alerts.StatusCode, $alerts.StatusDescription
+    exit 1
+}
+$parsedAlerts = $alerts.content | ConvertFrom-Json
+
+#Region ADD TEMPORARY SUPPORT FOR Non-Database Provider for Dependency Alerts in Public Preview, this will migrate to Database Provider in the near future
```

Comment:
Will test this today, might be able to remove this section now that Dependency scan is now running using the DB provider

Reply:
tested and removed with [12fabe6](https://github.com/microsoft/GHAzDO-Resources/pull/8/commits/12fabe6d670ffdd52a82c8dd0d5ff4b756f16143)